app/views/spree/admin/data_feeds/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :page_title do %>
+  <%= Spree.t('admin.data_feeds.data_feeds') %>
+<% end %>
+
+<% content_for :page_actions do %>
+  <%= button_link_to Spree.t('admin.data_feeds.new_data_feed'), new_object_url, class: "btn-success", icon: 'add.svg', id: 'admin_new_data_feed_link' %>
+<% end if can?(:create, Spree::DataFeed) %>
+
+<% if @data_feeds.any? %>
```

Comment:
Isn't `#present?` more suitable here? `#any?` suggests that `@data_feeds` could be a non-empty array of falsy values.
```suggestion
<% if @data_feeds.present? %>
```

Reply:
Yeah, good point!